server/manager.js

```diff
@@ -69,6 +72,23 @@ exports.ClockManager = class extends EventEmitter {
       })
     }
 
+    this._timeSaver.getTimeFromFile().then(time => {
+      if (time) {
+        this._status = RUNNING
+
+        const currentTime = new Date()
+        const timeRun = Math.floor((currentTime.getTime() - time.getTime()) / 1000)
+        const matchTimeWithBuffer = MATCH_TIME + MATCH_TIME_BUFFER
+
+        if (timeRun < matchTimeWithBuffer) {
```

Comment:
Does this code implement this?
>The idea of the extra time was this:
If the clock crashes somewhere in the middle of the match, we resume the countdown when it comes up.
But what happens if the clock crashes 1 sec before end? By the time clock restarts, the 2:30 is over there won't be an end sound.
Solution: Up till x secs after 2:30, if the time file is still there when clock restarts, play the "end" before deleting time file.